ci/build_python.sh

```diff
@@ -9,11 +9,20 @@ export CMAKE_GENERATOR=Ninja
 
 rapids-print-env
 
+package_name="rmm"
+package_dir="python"
+
+version=$(rapids-generate-version)
+commit=$(git rev-parse HEAD)
+
+echo "${version}" | tr -d '"' > VERSION
```

Comment:
Why do we have to strip quotes here? Can we just not add quotes in `rapids-generate-version` instead? We repeat the quote-stripping logic at least twice in this PR.

Reply:
@ajschmidt8 was going to look into whether we could fix the tool to avoid including quotes.

Reply:
I fixed this in https://github.com/rapidsai/gha-tools/pull/84